### DISCUSSION TITLE
Discord as a Reddit alternative

### DISCUSSION THREAD

**Discussion by futurejohn** [GeorgeSG/sub.rehab#6] [General]
I've seen some discussion on Lemmy/KBin that Discord may not be the best substitute for Reddit. Mainly because Discord centrally manages all the servers, but also the feature set doesn't entirely match Reddit's. The centralization may lead to conflicts with the community in the future. Any thoughts on this?

[Original post](https://lemmy.world/post/293826)

[Discord-specific comment thread](https://lemmy.world/comment/349112)

**Comment by GeorgeSG** [comment]
I posted some thoughts [here](https://lemmy.world/comment/355335)

Excerpt: 

> I understand how Discord is not the same as the fediverse and that the interactions there are different. But my goal is to show where subreddits migrated to and what are viable alternatives. It’s a fact that a lot of subreddits choose to list Discord servers as official alternatives and I think I’d be misrepresenting what’s happening if I don’t include them.
>
>I plan some improvements on the filtering controls so hopefully that will help :)

My initial objective is not to influence/bias the migration of communities, but to document it.

**Reply by futurejohn** [reply]
Gotcha, I totally understand that. The filters definitely help and are a great compromise. Thank you for your work on this!
